src/virtual-machine.js

```diff
@@ -554,10 +567,11 @@ class VirtualMachine extends EventEmitter {
     /**
      * Create a variable by name.
      * @todo this only creates global variables by putting them on the stage
-     * @param {string} name The name of the variable
+     * @param {string} id The id of the variable.
+     * @param {string} name The name of the variable.
      */
-    createVariable (name) {
```

Comment:
Just wanted to point out that we should file an issue for any external API changes before we make them so we do not lose track of this, as it'll break the GUI. 

Come to think of it, this function was actually only being used as a kind of crutch to wire together the blocks -> gui -> VM. Now that we have the events coming directly to the VM through the listener we can probably get rid of this and the accompanying call from the GUI. Feel free to remove this function entirely!